src/game/APISocket.cpp

```diff
@@ -169,10 +169,8 @@ void CAPISocket::Release() {
     }
 */
 
-    for (int i = 0; i < 255; i++) {
-        memset(m_Statistics_Send_Sum, 0, sizeof(m_Statistics_Send_Sum));
-        memset(m_Statistics_Recv_Sum, 0, sizeof(m_Statistics_Recv_Sum));
-    }
+    memset(m_Statistics_Send_Sum, 0, sizeof(m_Statistics_Send_Sum));
+    memset(m_Statistics_Recv_Sum, 0, sizeof(m_Statistics_Recv_Sum));
```

Comment:
lol! this should be written in history books. How and who would've write this code. Thanks for updating this as well.